src/Activity/TrainWithExample/index.js

```diff
@@ -23,13 +23,11 @@ class TrainWithExample extends React.Component {
   recordExampleActivity = userAnswer => {
     console.log(this.props.sessionRef);
     this.newActivityRef.child("activity_type").set("example");
-    let image = parallelogramData[this.state.index].src;
-    if (this.props.activityChosen === "mammals"){
-      image = mammalsData[this.state.index].src;
-      this.newActivityRef.child("topic").set("mammals")
-    }else{
-      this.newActivityRef.child("topic").set("parallelograms");
-    }
+    const image = {
+      mammals: mammalsData[this.state.index].src,
+      parallelogram: parallelogramData[this.state.index].src 
+    }[this.props.activityChosen]
+    this.newActivityRef.child("topic").set(this.props.activityChosen);
```

Comment:
I can show you the database later so that you understand the change I suggest for this